### PR TITLE
Check Zabbix return status and fix "Invalid response header received" when using the bulk API

### PR DESCRIPTION
**Pull request by micha137**

### CODE REVIEW

File: bin/zabbix-sender [hunk 1]
```diff
@@ -4,6 +4,7 @@
 use strict;
 use warnings;
 
+use lib 'lib';
```
Comment (gittex) [inline bin/zabbix-sender]: I don't want to mess with the lib path. Why do you need this?

Comment (micha137) [inline bin/zabbix-sender]: I needed it for development to have bin/zabbix-sender use my Zabbix/sender.pm from development.  You can leave out this line.

File: lib/Zabbix/Sender.pm [hunk 2]
```diff
@@ -201,6 +212,17 @@ sub _encode_request {
     return $output;
 }
 
+sub _check_info {
```
Comment (gittex) [inline lib/Zabbix/Sender.pm]: Please add a little documentation on what this method does.

Comment (micha137) [inline lib/Zabbix/Sender.pm]: OK, added with the next commits.

**Comment by gittex** [comment]
Looks good now. Thanks!

**Pull request merged**